src/common/message-types.ts

```diff
@@ -1,19 +1,37 @@
 export default class MessageTypes {
-  public static GQL_INIT = 'init';
-  public static GQL_INIT_SUCCESS = 'init_success';
-  public static GQL_INIT_FAIL = 'init_fail';
-  public static GQL_KEEP_ALIVE = 'keep_alive';
-  public static GQL_TERMINATE = 'terminate';
+  public static GQL_CONNECTION_INIT = 'connection_init';
+  public static GQL_CONNECTION_ACK = 'connection_ack';
+  public static GQL_CONNECTION_ERROR = 'connection_error';
+  // NOTE: This one here don't follow the standard due to connection optimization
+  public static GQL_CONNECTION_KEEP_ALIVE = 'ka';
+  public static GQL_CONNECTION_TERMINATE = 'connection_terminate';
 
+  // NOTE: Those message types are deprecated and will be removed soon.
+  /**
+   * @deprecated
+   */
   public static GQL_SUBSCRIPTION_START = 'subscription_start';
+  /**
+   * @deprecated
+   */
   public static GQL_SUBSCRIPTION_DATA = 'subscription_data';
+  /**
+   * @deprecated
+   */
   public static GQL_SUBSCRIPTION_SUCCESS = 'subscription_success';
+  /**
+   * @deprecated
+   */
   public static GQL_SUBSCRIPTION_FAIL = 'subscription_fail';
+  /**
+   * @deprecated
+   */
   public static GQL_SUBSCRIPTION_END = 'subscription_end';
 
-  public static GQL_REQUEST = 'request';
-  public static GQL_REQUEST_DATA = 'request_data';
-  public static GQL_REQUEST_FAIL = 'request_fail';
+  public static GQL_START = 'start';
+  public static GQL_DATA = 'data';
+  public static GQL_ERROR = 'error';
+  public static GQL_COMPLETE = 'complete';
```

Comment:
still missing "stop"/"cancel" -> client message that indicates 'we dont need this data anymore'